File: app/src/main/java/com/github/pockethub/android/ui/item/code/PathHeaderItem.java
package com.github.pockethub.android.ui.item.code;

import android.support.annotation.NonNull;
import android.text.method.LinkMovementMethod;
import android.view.View;
import android.widget.TextView;

import com.github.pockethub.android.R;
import com.github.pockethub.android.ui.item.BaseViewHolder;
import com.xwray.groupie.Item;

import butterknife.BindView;

public class PathHeaderItem extends Item<PathHeaderItem.ViewHolder> {

    private CharSequence path;

    public PathHeaderItem(CharSequence path) {
        super(path.hashCode());
        this.path = path;
    }

    @Override
    public void bind(@NonNull ViewHolder viewHolder, int position) {
        viewHolder.path.setText(path);
    }

    @Override
    public int getLayout() {
        return R.layout.path_item;
    }

    @NonNull
    @Override
    public ViewHolder createViewHolder(@NonNull View itemView) {
        return new ViewHolder(itemView);
    }

    public class ViewHolder extends BaseViewHolder {
        @BindView(R.id.tv_path) TextView path;

        public ViewHolder(@NonNull View rootView) {
            super(rootView);
            path.setMovementMethod(LinkMovementMethod.getInstance());
        }
    }
}

File: app/src/main/java/com/github/pockethub/android/ui/item/code/PathHeaderItem.kt
package com.github.pockethub.android.ui.item.code

import com.github.pockethub.android.R
import com.xwray.groupie.kotlinandroidextensions.Item
import com.xwray.groupie.kotlinandroidextensions.ViewHolder
import kotlinx.android.synthetic.main.path_item.*

class PathHeaderItem(private val path: CharSequence) : Item(path.hashCode().toLong()) {

    override fun getLayout() = R.layout.path_item

    override fun bind(holder: ViewHolder, position: Int) {
        holder.tv_path.text = path
    }
}
